src/index.ts

```diff
@@ -678,192 +511,82 @@ class NixInstallerAction extends DetSysAction {
     }
   }
 
-  async spawnDockerShim(): Promise<void> {
+  async spawnDoubleFork(): Promise<void> {
```

Comment:
This isn't really a double fork, so let's just give it a more accurate name.
```suggestion
  async spawnDetached(): Promise<void> {
```